docs/Projects/TAD/ADRs/0003-tad-tool.md

```diff
@@ -0,0 +1,52 @@
+# ADR-0003 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+We are considering tooling for organizations to get more grip on their algorithms.
+Tooling for, for instance bias and fairness tests, and assessments (like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)).
+
+Transparency, we think, can be fostered by sharing reports from such a tool in a [standardized way](../Reporting-Standard/latest.md).
+
+There are several existing open source
+[tools](../Existing-Tools/Comparison/tools/) which
+we [have assessed](../Existing-Tools/Comparison/tools_comparison/).
+Some support only assessments, others already combine more features and can generate a report.
+There is however no tool
+that supports all the [requirements](../Existing-Tools/Comparison/requirements/) we have.
+
+These are our main requirements:
```

Comment:
What do we think of adding a requirement that the tool can provide an overview of multiple AI systems? That it can be used as an internal register for algorithms?

Reply:
I think that is a good idea. Should it be:
- multiple projects with 1 algorithm each
- multiple projects with multiple algorithms
and if the second, does it re-use components (like an IAMA applies to 2 algorithms), but it does generate 2 system cards?